### ISSUE TITLE
Threads.java:43-45:...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/cactoos#1000]
The puzzle `972-6ba919a9` from #972 has to be resolved: 

https://github.com/yegor256/cactoos/blob/c425c369fce5dac1fb7b019d531303d4149fa8e2/src/main/java/org/cactoos/experimental/Threads.java#L43-L45

The puzzle was created by @dgroup on 12-Dec-18. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#1000](https://github.com/yegor256/cactoos/issues/1000) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1000](https://github.com/yegor256/cactoos/issues/1000) assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/cf7fe979-ba43-4b96-a549-d745f2fc202f); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by llorllale** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/1000#issuecomment-474140220))

@llorllale The user @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#1000](https://github.com/yegor256/cactoos/issues/1000), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
The job [#1000](https://github.com/yegor256/cactoos/issues/1000) assigned to @golyalpha[/z](https://www.0crat.com/u/golyalpha), here is [why](https://www.0crat.com/footprint/C63314D6Z/e6b154bd-7e1a-41cf-8d39-36808418a831); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
The architect of the project has changed; @llorllale[/z](https://www.0crat.com/u/llorllale) is not at this role anymore; @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) is the architect now

**Comment by golyalpha** [comment]
@0crat refuse

**Comment by golyalpha** [comment]
@0crat resign

**Comment by 0crat** [comment]
> @0crat resign ([here](https://github.com/yegor256/cactoos/issues/1000#issuecomment-567504138))

@golyalpha The user @golyalpha[/z](https://www.0crat.com/u/golyalpha) resigned from [#1000](https://github.com/yegor256/cactoos/issues/1000), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged (job refused in 6304 hours), see [§6](http://www.zerocracy.com/policy.html#6): -20 point(s) just awarded to @golyalpha[/z](https://www.0crat.com/u/golyalpha)

**Comment by 0crat** [comment]
The job [#1000](https://github.com/yegor256/cactoos/issues/1000) assigned to @iakunin[/z](https://www.0crat.com/u/iakunin), here is [why](https://www.0crat.com/footprint/C63314D6Z/4c8cbe21-4816-4bbf-9de8-198c28ab579d); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1000` already assigned to @iakunin, can't assign to @iakunin

**Comment by iakunin** [comment]
@0crat wait for https://github.com/yegor256/cactoos/pull/1267 to be merged

**Comment by 0crat** [comment]
> @0crat wait for https://github.com/yegor256/cactoos/pull/1267 to be merged ([here](https://github.com/yegor256/cactoos/issues/1000#issuecomment-569404679))

@iakunin The impediment for [#1000](https://github.com/yegor256/cactoos/issues/1000) was registered successfully by @iakunin[/z](https://www.0crat.com/u/iakunin)

**Comment by 0pdd** [comment]
The puzzle `972-6ba919a9` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1277](https://github.com/yegor256/cactoos/issues/1277) is still not solved.

**Comment by 0crat** [comment]
@iakunin[/z](https://www.0crat.com/u/iakunin) resigned from [#1000](https://github.com/yegor256/cactoos/issues/1000), since the job is not in scope anymore

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@2ea4004cf7d7, thread: PQ-C63314D6Z
com.zerocracy.claims.ClaimIn[214] java.lang.IllegalArgumentException: Parameter "age" not found in "Order was canceled" among: [reason, priority, job, cause, _parent_sig, flow, voluntarily, login]
```

[0.51.1](https://github.com/zerocracy/farm): CID: [1a3472ba-4361-4ac1-9be2-483b83183333](https://www.0crat.com/footprint/C63314D6Z/1a3472ba-4361-4ac1-9be2-483b83183333), Type: "Order was canceled"

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1406](https://github.com/yegor256/cactoos/issues/1406), [#1407](https://github.com/yegor256/cactoos/issues/1407) are still not solved; solved: [#1277](https://github.com/yegor256/cactoos/issues/1277).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1406](https://github.com/yegor256/cactoos/issues/1406) is still not solved; solved: [#1277](https://github.com/yegor256/cactoos/issues/1277), [#1407](https://github.com/yegor256/cactoos/issues/1407).

**Comment by 0pdd** [comment]
@0pdd all 3 puzzles are solved here: [#1277](https://github.com/yegor256/cactoos/issues/1277), [#1406](https://github.com/yegor256/cactoos/issues/1406), [#1407](https://github.com/yegor256/cactoos/issues/1407).
